docs/source/platform/schema-validation.md

```diff
@@ -131,15 +131,37 @@ Of course, it’s also possible to leave the field in place indefinitely!
 
 <h2 id="cli">Checking schema changes with the Apollo CLI</h2>
 
-To check and see the difference between the current published schema and a new version, run the following command, substituting the appropriate GraphQL endpoint URL and an API key:
+To check and see the difference between the current published schema and a new version, make sure you have a valid [service configuration](../resources/apollo-config.html#service-config) like so, substituting the appropriate GraphQL endpoint URL and an API key:
```

Comment:
Reading through the doc now, this lead-in sentence is surprising with the preceding header(the original content definitely pushed in this direction just happened to notice with your change 😄). In my mind, there is something missing: an indication that this is a setup section in the title or an explanation of the check/cli's utility or both. 

The explanation of the cli's effect at the end of this section would probably provide a nice motivation with some rewording before diving into the setup, maybe indicated with a subheader.